tests/API/XCCDF/unittests/test_remediation_invalid_characters.xccdf.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Benchmark xmlns="http://checklists.nist.gov/xccdf/1.2" id="xccdf_moc.elpmaxe.www_benchmark_test">
+  <status>accepted</status>
+  <version>1.0</version>
+  <Rule selected="true" id="xccdf_moc.elpmaxe.www_rule_1">
+    <title>Ensure that file exists and it is not executable</title>
+    <fix system="urn:xccdf:fix:script:sh">
+      echo -e "\a \b \r \x1F"
+      touch test_file
```

Comment:
I think, you should remove this file using Makefile.am, because it left there. We will see it in jenkins result.